blues/application/deploy.py

```diff
@@ -164,6 +164,9 @@ def install_virtualenv():
     with sudo_project():
         virtualenv.create(virtualenv_path())
 
+    with virtualenv.activate(virtualenv_path()):
```

Comment:
Maybe we should do this outside the virtualenv too?

Reply:
@jocke-l It happens both on the system and within the venv, since I changed this line: https://github.com/5monkeys/blues/pull/106/files#diff-de31ff49f799c7d793f2a68765861eeeR91

Reply:
Ah, I missed that one :)